src/app_engine/apprtc.py

```diff
@@ -590,6 +590,20 @@ def get(self):
     params = get_room_parameters(self.request, None, None, None)
     self.response.write(json.dumps(params))
 
+def checkIfRedirect(self):
+  REDIRECT_DOMAINS = [
+    'apprtc.appspot.com', 'apprtc.webrtc.org', 'www.appr.tc'
```

Comment:
I guess we forgot to include www. in the certificate. Not sure if this is important enough to warrant us to create a new certificate. @juberti @samdutton  WDYT?